arches/install/arches-templates/project_name/urls.py-tpl

```diff
@@ -2,9 +2,14 @@ from django.conf import settings
 from django.conf.urls.static import static
 from django.conf.urls.i18n import i18n_patterns
 from django.urls import include, path
+from arches.app.views.auth import PasswordResetView
 
 urlpatterns = [
     path('', include('arches.urls')),
+    path(

```

Comment:
Can this path go into `arches.urls.py` instead?

Reply:
It requires there to be an html version of the password reset email which is not in out of the box Arches - if we put this in arches.urls.py wouldn't it cause a problem?

Reply:
That's a good point.

@apeters , @robgaston  or @chiatt : What would be the right move here? Duplicate these templates in arches-core so that people run un-projected arches can use the feature and the url can be moved, or leave these changes as-is at the project level?

Reply:
@chrabyrd I think this is the last outstanding issue with the pull request.  Has a decision been reached on this?

Reply:
:wave: @csmith-he 

Not yet, but I'm getting the discussion rolling and should have an answer for you ASAP

Reply:
@chrabyrd I think arches should come with a default template for the password reset out of the box.

Reply:
👋 @chrabyrd I updated my previous comment.

Reply:
@apeters perfect, thanks for guidance! Also re: unprojected Arches, it's possible and I think Mike Fischer is doing it ( or at least he was ) . These changes defacto support it but I'm not saying we should explicitly put cycles to supporting it.

@csmith-he yes please move these changes into arches-core